CHANGELOG.md

```diff
@@ -1,3 +1,137 @@
+# 0.26.0 (2018-08-??)
+
+## Language changes
+
+- **(breaking-change)** Revert do not collapse unions for sibling types. ([#6351], thanks @asterite)
+- **(breaking-change)** Restrict `exp.@var` to types in the same hierarchy. ([#6358], thanks @asterite)
+- **(breaking-change)** Constant lookup context in macro is now lexical. ([#5354], thanks @MakeNowJust)
+- **(breaking-change)** Evaluate instance var initializers at the metaclass level (ie: disallow using `self`). ([#6414], thanks @asterite)
+- **(breaking-change)** Add `//` operator parsing. NB: No behaviour is assigned to this operator yet. ([#6470], thanks @bcardiff)
+- Add `&+` `&-` `&*` `&**` operators parsing. NB: No behaviour is assigned to these operators yet. ([#6329], thanks @bcardiff)
+- Add support for empty `case` without `when`. ([#6367], thanks @straight-shoota)
+
+### Macros
+
+- Add `pp!` and `p!` macro methods. ([#6374], [#6476], thanks @straight-shoota)
+
+## Standard library
+
+### Numeric
+
+- Fixed `Random#rand(Range(Float, Float))` to return `Float`. ([#6445], thanks @straight-shoota)
+- Add docs of big module overloads. ([#6336], thanks @laginha87)
+
+### Text
+
+- **(breaking-change)** `String#from_utf-16(pointer : Pointer(UInt16))` returns now `{String, Pointer(UInt16)}`. ([#6333], thanks @straight-shoota)
```

Comment:
One grep to much ;)

Reply:
Am I missing something or you are pointing the fact that is was duplicated?